src/app/pages/CChainPages/LatestBlocksAndTransactionsList/TransactionsList.tsx

```diff
@@ -64,7 +65,8 @@ const TransactionsList: FC<TransactionsListProps> = ({ title, items, to, link })
                 fontWeight="fontWeightBold"
                 sx={{ color: 'error.main', alignSelf: 'flex-end', my: '.5rem' }}
             >
-                Hover on tx value to view full exact amount.
+                Transaction values are approximate, hover on number or click on transaction to view
```

Comment:
`hover over the number` would be more appropriate wording, also, should the line break be there at the end?

Reply:
The wording has been provided by the product team @tabeaarnold.